upi/vsphere/machine/ignition.tf

```diff
@@ -20,23 +19,23 @@ data "ignition_file" "hostname" {
 data "ignition_file" "static_ip" {
   count = "${var.instance_count}"
 
-  filesystem = "root"
-  path       = "/etc/sysconfig/network-scripts/ifcfg-ens192"
-  mode       = "420"
+  path       = "/etc/NetworkManager/system-connections/eth0"
+  mode       = "0600"
 
   content {
     content = <<EOF
-TYPE=Ethernet
-BOOTPROTO=none
-NAME=ens192
-DEVICE=ens192
-ONBOOT=yes
-IPADDR=${local.ip_addresses[count.index]}
-PREFIX=${local.mask}
-GATEWAY=${local.gw}
-DOMAIN=${var.cluster_domain}
-DNS1=1.1.1.1
-DNS2=9.9.9.9
+[connection]
+id=Wired connnection 1
+uuid=27afa607-ee36-43f0-b8c3-9d245cdc4bb3
+type=802-3-ethernet
+interface-name=eth0
+autoconnect=true
+
+[ipv4]
+method=manual
+dns=1.1.1.1;9.9.9.9
```

Comment:
not for this PR, more like a todo to inject the DNS from a var rather than hardcoded

Reply:
These are sample terraform files in any case, users may adjust those if needed anyway